test/unit/FileUtilsTest.cpp

```diff
@@ -39,6 +39,8 @@
 
 #include "Support.hpp"
 
+#include <iostream>
```

Comment:
Grrr, I should have removed it before updating the commit.
I will wait for the CI-s first, then remove it.